### PR TITLE
NMS-9303: Remove unused and dead event definitions

### PR DESCRIPTION
**Pull request by indigo423**
Removed unused and dead event definitions from configuration and event constants.

* JIRA: http://issues.opennms.org/browse/NMS-9303

### CODE REVIEW

**Comment by indigo423** [comment]
@agalue Splitted the cleanup for unused and dead events in a separate pull request. The renaming of the left overs which are handled by Discovery-Daemon, SNMP Interface Poller and Provisiond are will be opened in a separate PR. I think this makes it much easier to review.

**Comment by indigo423** [comment]
Review hint: The main driver for removing the events was the EventConstants.java and Provisioner.java with the FIXME/TODO Unsupported methods.
* https://github.com/OpenNMS/opennms/pull/1450/files#diff-9e903204dc4da122ab079bdb08eb1f47
* https://github.com/OpenNMS/opennms/pull/1450/files#diff-ce2a59865a31eb0c56f380cc44606f7e

**Comment by indigo423** [comment]
@RangerRick Do you know how to add the GitHub bamboo integration to this branch? It is for some reasons missing.

**Comment by RangerRick** [comment]
Might be there was a network burp while it was being started.  Not sure you can other than doing another push and/or restarting the build in Bamboo.

**Comment by indigo423** [comment]
@RangerRick ah ok, thx for info

**Comment by RangerRick** [comment]
Ah, Bamboo is currently building hash `4e83041ecccff67047d48e0f8fef900bc20b18ac` but this PR is currently topped by `0d4dc717615ae027b6b63da065fe2a16e1daec77` so they don't match up.  If you go into the "Commits" tab in the PR here, you'll see the `4e8...` one has a yellow dot next to it because it's in-progress.

You can kill your bamboo build, or just wait for the next one to trigger.

**Comment by indigo423** [comment]
@RangerRick stopped the build and trigger a current one. He picked up the next commit and works like a charm. Thanks for help

File: opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew-20.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+[[releasenotes-19]]
```
Comment (j-white) [inline opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew-20.adoc]: 19 vs 20

**Comment by j-white** [comment]
:+1: from me.

**Pull request merged**